branchff

```diff
@@ -116,6 +116,16 @@ logrun mkdir -p $WORKDIR
 gitlib::sync_repo $K8S_GITHUB_SSH $TREE_ROOT || common::exit 1 "Exiting..."
 logrun cd $TREE_ROOT
 
+# Is the master branch in a reasonable state to fast forward over the branch?
+GIT_DESCRIBE=$(git describe)
+if [[ ! $GIT_DESCRIBE =~ ^v${RELEASE_BRANCH/release-/} ]]; then
```

Comment:
Once the release branch is cut, master immediately gets tagged with alpha.0 for the next minor. Maybe we should enforce that the latest tag on master has to be `X.(Y+1).0-alpha.0`? If you add `--abrev=0` it can be an exact match.